### PR TITLE
My Prime_E Keymap

### PR DESCRIPTION
**Pull request by ifohancroft**
## Description

Just submitting my Prime_E keymap for inclusion.

## Types of Changes

- [x] Keymap/layout/userspace (addition or update)
- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [ ] Documentation

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.

### CODE REVIEW

File: keyboards/primekb/prime_e/keymaps/ifohancroft/keymap.c [hunk 1]
```diff
@@ -0,0 +1,76 @@
+#include QMK_KEYBOARD_H
```
Comment (spidey3) [inline keyboards/primekb/prime_e/keymaps/ifohancroft/keymap.c:1]: Could you add a GPL2+ compatible license header here? 

For instance: 
https://github.com/qmk/qmk_firmware/blob/28ff51175b6d45ce5a8dbfe7ed6e7a12df3bc8a8/quantum/template/base/keyboard.c#L1-L15

**Review by spidey3 — CHANGES_REQUESTED** [review]

**Comment by ifohancroft** [comment]
Done. Sorry, I totally forgot about that.

Perhaps adding a license header template to the keymap documentation page is worth it?

**Review by spidey3 — APPROVED** [review]

File: keyboards/primekb/prime_e/keymaps/ifohancroft/config.h [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#pragma once
+
+// place overrides here
+#undef USB_POLLING_INTERVAL_MS
```
Comment (drashna) [inline keyboards/primekb/prime_e/keymaps/ifohancroft/config.h:21]:
```suggestion
```

**Review by drashna — CHANGES_REQUESTED** [review]

File: keyboards/primekb/prime_e/keymaps/ifohancroft/config.h (hunk 2 above)
Comment (ifohancroft) [reply keyboards/primekb/prime_e/keymaps/ifohancroft/config.h:21]: I don't get it?
Don't I have to undefine USB_POLLING_INTERVAL_MS before defining it again?

Comment (ifohancroft) [reply keyboards/primekb/prime_e/keymaps/ifohancroft/config.h]: Done

File: keyboards/primekb/prime_e/keymaps/ifohancroft/config.h [hunk 3]
```diff
@@ -0,0 +1,5 @@
+#pragma once
```
Comment (drashna) [inline keyboards/primekb/prime_e/keymaps/ifohancroft/config.h:1]: Could you add a GPL2+ compatible license header here? 

For instance: 
https://github.com/qmk/qmk_firmware/blob/28ff51175b6d45ce5a8dbfe7ed6e7a12df3bc8a8/quantum/template/base/keyboard.c#L1-L15

**Review by drashna — CHANGES_REQUESTED** [review]

File: keyboards/primekb/prime_e/keymaps/ifohancroft/config.h (hunk 3 above)
Comment (ifohancroft) [reply keyboards/primekb/prime_e/keymaps/ifohancroft/config.h:17]: Done

**Review by fauxpark — APPROVED** [review]

**Pull request merged**